packages/teams-js/test/public/runtime.spec.ts

```diff
@@ -190,107 +190,226 @@ describe('runtime', () => {
     return result;
   }
 
-  function generateVersionNumberOlderThanGivenVersion(version: string): string {
-    return `0.${version}`;
-  }
-
   describe('generateVersionBasedTeamsRuntimeConfig tests based on Teams default configuration', () => {
-    Object.entries(mapTeamsVersionToSupportedCapabilities).forEach(([version, capabilityAdditionsForEachVersion]) => {
-      capabilityAdditionsForEachVersion.forEach((capabilityAdditionsForClientTypesInASpecificVersion) => {
-        const capabilityAdditionsForThisVersion = capabilityAdditionsForClientTypesInASpecificVersion.capability;
-        capabilityAdditionsForClientTypesInASpecificVersion.hostClientTypes.forEach((clientType) => {
-          it(`Back compat host client type ${clientType} supporting up to ${version} should support ${JSON.stringify(
-            capabilityAdditionsForThisVersion,
-          )}`, async () => {
+    it('Validate that all client types where a capability began to be supported report that capability IS supported in that version', async () => {
```

Comment:
Do you think it would be worth restructuring this so instead of one big "super" test for all versions/capabilities/etc. we had more discrete tests for smaller numbers of combinations? We do something likes this for testing framecontexts:

![image](https://github.com/OfficeDev/microsoft-teams-library-js/assets/18446904/42003132-ccb3-49b4-99a1-dd4ce34e8568)

Doing this might make the tests take longer (depending on how many they are), but could also make test failures easier to understand and investigate

Reply:
Your suggestion is interesting; I would like to explore it further. If it's okay with you, I'd like to check in the change as-is to unblock @maglims to unblock Teams Mobile. I will follow-up in a separate PR on the possibilities of further restructuring.